Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -80,6 +80,7 @@ The main active response playbook is the `Cortex ASM - ASM Alert` playbook. This
   - [Cortex ASM - Qualys Enrichment](#cortex-asm---qualys-enrichment)
   - [Cortex ASM - Rapid7 Enrichment](#cortex-asm---rapid7-enrichment)
   - [Cortex ASM - Remediation Confirmation Scan](#cortex-asm---remediation-confirmation-scan)
+  - [Cortex ASM - Remediation Objectives](#cortex-asm---remediation-objectives)
```

Comment:
Please move below `Cortex ASM - Remediation Path Rules`.  Should be alphanumeric order, same goes for the header section it links to.

Reply:
[change README order](https://github.com/demisto/content/pull/27795/commits/9a52179db635f2ca4f25e1331c7fa652cc283884)